docs/rules/use-class-secure-random.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Use Class SecureRandom
+ruleId: UseClassSecureRandom
+since: 3.20.0
+minJavaVersion: 1.2
+remediationCost: 5
+links:
+    - displayName: "S2245"
+      url: "https://sonarcloud.io/organizations/default/rules?open=java%3AS2245&q=pseudorandom"
+    
+description:
+    This rule prevents vulnerabilities due to a pseudo-random number generator (PRNGs) by replacing it by a cryptographically strong random number generator (RNG).
+tags: ["Java 1.2", "Security"]
+---
+
+# Use Class SecureRandom
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
```

Comment:
Since this issue is listed as "Critical" in Sonarcloud (as default), we should mention that in the description. This highlights the importance of this rule.